src/Appium.Net/Appium/AppiumDriver.cs

```diff
@@ -75,15 +75,37 @@ public AppiumDriver(AppiumLocalService service, ICapabilities appiumOptions)
         }
 
         public AppiumDriver(Uri remoteAddress, ICapabilities appiumOptions, TimeSpan commandTimeout)
-            : this(new AppiumCommandExecutor(remoteAddress, commandTimeout), appiumOptions)
+            : this(remoteAddress, appiumOptions, DefaultCommandTimeout, AppiumClientConfig.DefaultConfig())
```

Comment:
@KazuCocoa This seems to have broken being able to specify your own command timeout in 5.0.0-rc.1, which pretty much breaks Saucelabs because their real devices can take longer than 60 seconds to be provisioned.

Reply:
I already fixed this one on another PR. I need to release a new version